temporal-serviceclient/src/main/java/io/temporal/serviceclient/ChannelManager.java

```diff
@@ -293,7 +287,7 @@ public void connect(String healthCheckServiceName, @Nullable Duration timeout) {
         new GrpcRetryer.GrpcRetryerOptions(
             RpcRetryOptions.newBuilder().setExpiration(timeout).validateBuildWithDefaults(), null);
 
-    new GrpcRetryer(getServerCapabilities())
+    new GrpcRetryer(getServerCapabilitiesNow())
```

Comment:
Does this change turn server capabilities from lazy to eager?

Reply:
Yes, as you requested.

Reply:
I don't think we should change standard client behavior, I just wanted to confirm we fetched them on worker start (sorry if I was unclear). I figure we probably do since some of those capabilities are needed for worker, but just wanted to confirm.